SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -0,0 +1,1109 @@
+%global security_hardening none
+%global debug_package %{nil}
+%define uname_r %{version}-%{release}
+
+%ifarch x86_64
+%define arch x86_64
+%define archdir x86
+%define config_source %{SOURCE1}
+%endif
+
+Summary:        Linux Kernel for Kata UVM
+Name:           kernel-uvm
+Version:        5.15.48.1
+Release:        6%{?dist}
```

Comment:
Nit: Release

Reply:
Thanks for catching this! Fixed in newest push